### PR TITLE
done with Javascript exercises

### PR DESCRIPTION
**Pull request by codewithmanny**
I am done with both javascript exercises and attached is a picture of the output.

JS w3schools

javascript exercise

### CODE REVIEW

File: emmanuelagyei/js/variable.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+let y =2 
+let firstName = "Emmanuel";
+let lastName = "Agyei"
+
+let fullName = firstName +  lastName;
+
+console.log(fullName);
+
+console.log(fullName + y);
+
```
Comment (rbongiorno-ey) [inline emmanuelagyei/js/variable.js:16]: The requirement here is to multiply y by 2 and output the result.

File: emmanuelagyei/js/variable.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+// Write a JavaScript program to find a value which is nearest to 100 from two different given integer values. 
+// Example Test cases:  
+// findNearestValueTo100(80, 99) should result in 99 
+// findNearestValueTo100(-10, -9) should result in -9 
+
+function nearest (a,b){
+let a1 = 100-a 
+let b1 = 100-b
+
```
Comment (rbongiorno-ey) [inline emmanuelagyei/js/variable.js:26]: You should be looking for the absolute difference Math.abs(n - 100).

File: emmanuelagyei/js/variable.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+if (a1>b1){
+    return b + "is nearest"
+}else {
+    return a + "is nearest"
```
Comment (rbongiorno-ey) [inline emmanuelagyei/js/variable.js:30]: You might want to check for a1==b1, at which a _and_ b are the same distance from 100.

**Review by rbongiorno-ey — CHANGES_REQUESTED** [review]

File: emmanuelagyei/js/variable.js (hunk 2 above)
Comment (codewithmanny) [reply emmanuelagyei/js/variable.js:28]: fixed . please check line 26 and 27

**Review by madhanramu — APPROVED** [review]

**Review by rbongiorno-ey — APPROVED** [review]

**Pull request merged**